assignments/prototypes.js

```diff
@@ -14,97 +13,173 @@
   * dimensions
   * destroy() // prototype method -> returns the string 'Object was removed from the game.'
 */
+const GameObject = function(obj) {
+  this.createdAt = obj.createdAt;
+  this.dimensions = obj.dimensions;
+};
+GameObject.prototype.destroy = function() {
+  return `${this.name} was removed from the game.`;
+};
 
 /*
-  === CharacterStats ===
-  * hp
-  * name
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
+=== CharacterStats ===
+* hp
+* name
+* takeDamage() // prototype method -> returns the string '<object name> took damage.'
+* should inherit destroy() from GameObject's prototype
 */
 
+const CharacterStats = function(obj) {
+  GameObject.call(this, obj);
+  this.hp = obj.hp;
+  this.name = obj.name;
+};
+// console.log(GameObject.prototype);
+CharacterStats.prototype = {...GameObject.prototype};
```

Comment:
Nice implementation of ES6 syntax to copy objects properties, very creative method for implementing inheritance! 👍